wiki/Hit_object/Approach_circle/en.md

```diff
@@ -6,7 +6,7 @@ tags:
 
 # Approach circle
 
-*Approach circles* are coloured circles which shrink around [hit circles](/wiki/Hit_object/Hit_circle) in [osu!](/wiki/Game_mode/osu!) [beatmaps](/wiki/Beatmap). They help players determine when to click to earn the biggest amount of [score](/wiki/Gameplay/Score) by overlapping with the hit circle at the correct moment in a song. The time it takes for an approach circle to reach a hit circle is determined by a beatmap's [approach rate](/wiki/Beatmapping/Approach_rate) which is set by the mapper. The colour of approach circles is also determined by the combo colours used by the mapper of a beatmap.
+*Approach circles* are coloured circles which shrink around [hit circles](/wiki/Hit_object/Hit_circle) in [osu!](/wiki/Game_mode/osu!) [beatmaps](/wiki/Beatmap). They help players determine when to click to earn the biggest amount of [score](/wiki/Score) by overlapping with the hit circle at the correct moment in a song. The time it takes for an approach circle to reach a hit circle is determined by a beatmap's [approach rate](/wiki/Beatmapping/Approach_rate) which is set by the mapper. The colour of approach circles is also determined by the [combo colours](/wiki/Gameplay/Combo_colour) used by the mapper of a beatmap.
```

Comment:
no this would use a redirect

```suggestion
*Approach circles* are coloured circles which shrink around [hit circles](/wiki/Hit_object/Hit_circle) in [osu!](/wiki/Game_mode/osu!) [beatmaps](/wiki/Beatmap). They help players determine when to click to earn the biggest amount of [score](/wiki/Gameplay/Score) by overlapping with the hit circle at the correct moment in a song. The time it takes for an approach circle to reach a hit circle is determined by a beatmap's [approach rate](/wiki/Beatmapping/Approach_rate) which is set by the mapper. The colour of approach circles is also determined by the [combo colours](/wiki/Gameplay/Combo_colour) used by the mapper of a beatmap.
```